modules/logging-release-notes-5-6-20.adoc

```diff
@@ -0,0 +1,93 @@
+// module included in /logging/logging-5-6-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-6-20_{context}"]
+= Logging 5.6.20
+This release includes link:https://access.redhat.com/errata/RHBA-2024:3740[OpenShift Logging Bug Fix 5.6.20]
+
+[id="logging-release-notes-5-6-20-bug-fixes"]
+== Bug fixes
+
+* Before this update, there was a delay in restarting ingesters when configuring `LokiStack` as the {loki-op} set the write-ahead log `replay_memory_ceiling` to zero bytes for the `1x.demo` size. With this update, the minimum value used for the `replay_memory_ceiling` has been increased to avoid delays. (link:https://issues.redhat.com/browse/LOG-5617[LOG-5617])
```

Comment:
```suggestion
* Before this update, there was a delay in restarting ingesters when configuring `LokiStack`, because the {loki-op} set the write-ahead log `replay_memory_ceiling` to zero bytes for the `1x.demo` size. With this update, the minimum value used for the `replay_memory_ceiling` has been increased to avoid delays. (link:https://issues.redhat.com/browse/LOG-5617[LOG-5617])
```